dev-utils/test-servers.js

```diff
@@ -27,6 +27,7 @@ const express = require('express')
 const serveIndex = require('serve-index')
 const proxy = require('express-http-proxy')
 const { join } = require('path')
+const useragent = require('useragent')
```

Comment:
I'm not sure if we need to solve this issue on the server side, seems like this is something we can check on the browser.